.gitignore

```diff
@@ -3,7 +3,13 @@
 
 # ignore .env file with secret key, app config and database url
 .env
+.DS_Store
+.idea/
+token.pickle
```

Comment:
Store these in a [local gitignore](https://help.github.com/en/github/using-git/ignoring-files#create-a-local-gitignore) as they're specific to your setup.

I believe `token.pickle` is fine, but remove the first two.